ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ntp-settings-1.yml

```diff
@@ -23,6 +23,16 @@ ntp_settings:
   authentication_keys:
     - id: 1
       hash_algorithm: sha1
-      key: someobfuscatedkey
+      key: 06071D285F5A08
       key_type: 7
+      # Making sure `key` takes precedence over `cleartext_key`
+      cleartext_key: arista
```

Comment:
For OSPF we added `cleartext_simple_auth_key`, but NTP is using `cleartext_key`. The reason for the naming is clear and I'm NOT suggesting to change it, but we certainly need an issue for 6.0 to clean this. Looking at all this knobs makes me very sad...

Reply:
no

for OSPF we have both `key` under message digest and `simple_auth_key` because there are two different authentication type. Thats why